src/Analyzers/CSharp/Tests/AddInheritdoc/AddInheritdocTests.cs

```diff
@@ -304,9 +302,9 @@ public virtual void M() { }
             /// Some doc.
             public class Derived: BaseClass
             {
-                               /// <inheritdoc/>
                                // Comment 1
-              /* Comment 2 */  public /* Comment 3 */ override void M /* Comment 4 */ ()  /* Comment 5 */ { } /* Comment 6 */
+              /* Comment 2 */  /// <inheritdoc/>
+              public /* Comment 3 */ override void M /* Comment 4 */ ()  /* Comment 5 */ { } /* Comment 6 */
```

Comment:
this test case is pointless.  no one writes cod ethis way.  keeping things the same as they were before is not a desire here.